src/main/java/uk/gov/hmcts/cmc/claimstore/events/offer/OfferRespondedEvent.java

```diff
@@ -44,6 +46,6 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return ReflectionToStringBuilder.toString(this);
+        return ReflectionToStringBuilder.toString(this, ourStyle());
```

Comment:
why?

Reply:
it was wrong before, we have our own style for `ReflectionToStringBuilder` and everywhere else is using it